### ISSUE TITLE
Error, Notification: Errors and notifications content present under expand/collapse blade is not visible at 400% resize mode.

### ISSUE THREAD

**Issue by ato9000** [Azure/cosmos-explorer#1052]
https://msdata.visualstudio.com/CosmosDB/_workitems/edit/1245576

**Comment by ato9000** [comment]
Note: User credentials should NOT be included in the bug. 

Pre-Requisite:
Select "Core(SQL)" in API combo box while creating the Cosmos DB.
Go to Browser Settings->Zoom->400%.

Repro Steps:
Open URL: https://ms.portal.azure.com/ and login using valid credentials.
Navigate to "Azure Cosmos DB " from left pane and select it.
Navigate to any of the Azure Cosmos DB which is already created and select it.
Navigate to "Data Explorer" from the left and select it.
Navigate through all the controls under SQL API Section.
Create 'Sample Database'. 
Navigate to Expand/Collapsed button in the bottom and select it.
Navigate through all the controls like combo box, clear notifications.
Resize the page up to 400% and refresh the page.
Verify the issue.

Actual Result:
Errors and notifications content present under expand/collapse blade is not visible at 400% resize mode.

Expected Result:
Errors and notifications content and all controls present under expand/collapse blade should be visible at 400% resize mode.

![image](https://user-images.githubusercontent.com/8519407/132716141-5a64de9a-a178-4f74-938c-61f5ddbd122b.png)

![image](https://user-images.githubusercontent.com/8519407/132716165-1a035ced-cd29-4cc9-a651-df708a895a66.png)
